### PR TITLE
Colw/proposals graphql

### LINKED ISSUE #2955: Fetch Proposals Using GraphQL
**Is your feature request related to a problem? Please describe.**
With the new backend, it is now possible to retrieve Proposal information using GraphQL.

**Describe the solution you'd like**
Fetch lists of, and individual proposals using GraphQL.

### PR DESCRIPTION
**Pull request by colw**
Closes #2955

**Description:**

Fetch Proposal list using GraphQL

TODO: Fetch individual proposals

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/governance/LiProposal.vue [hunk 1]
```diff
@@ -9,46 +9,42 @@
...
 <script>
-import { mapState } from "vuex"
-import { getProposalStatus } from "scripts/proposal-status"
 export default {
   name: `li-proposal`,
+  filters: {
+    trim: function(text, length) {
+      return text.length > length ? text.substring(0, length) + `…` : text
+    },
+    lowerCase: s => s.toLowerCase()
```
Comment (jbibla) [inline src/components/governance/LiProposal.vue]: what is `s`? please try to avoid abbreviations.

File: src/components/governance/LiProposal.vue [hunk 2]
```diff
@@ -104,24 +100,26 @@ export default {
   border: 2px solid;
   padding: 2px 4px;
   border-radius: 0.25rem;
+  color: var(--grey);
+  border-color: var(--grey);
 }
 
-.proposal-status.red {
+.proposal-status.rejected {
```
Comment (jbibla) [inline src/components/governance/LiProposal.vue]: smart!

File: src/gql/index.js [hunk 3]
```diff
@@ -19,3 +19,28 @@ export const ValidatorProfile = gql`
 
 export const validatorProfileResultUpdate = data =>
   data.validatorProfiles ? data.validatorProfiles[0] : undefined
+
+export const allProposals = gql`
+  query allProposals {
```
Comment (jbibla) [inline src/gql/index.js]: this is amazing

**Comment by jbibla** [comment]
is there an issue associated with this PR?

**Comment by jbibla** [comment]
on page proposal, can you please switch `white-space: pre-wrap;` to `white-space: pre-line;`

i know this isn't graphql, but since you're in there 😄

**Comment by colw** [comment]
> on page proposal, can you please switch `white-space: pre-wrap;` to `white-space: pre-line;`
> 
> i know this isn't graphql, but since you're in there 😄

Yep. It's in the `TextBlock` component, and this is the only place it's used. Can do.

**Comment by colw** [comment]
> is there an issue associated with this PR?

Does not appear to be. Can make one.

**Comment by faboweb** [comment]
Let's close this? It's implemented by https://github.com/luniehq/lunie/pull/3022
I picked some of the nice ideas from here.

**Comment by colw** [comment]
Of course 👍

**Pull request closed without merging**